docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-the-latest-version-of-docker-sdk-integrated.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Is latest version of Docker SDK integrated and project is running on Docker SDK?
+description: This document allows you to assess if the latest version of Docker SDK is integrated in a project.
+template: howto-guide-template
+---
+
+# Is latest version of Docker SDK integrated and project is running on Docker SDK?
+
+{% info_block infoBox %}
+
+Resources: Backend
+
+{% endinfo_block %}
+
+## Description
+
+1. Search in the root of the project for the deploy files, they should have the following naming structure `deploy.(projectname)-(environmentname).yml`.
+2. Ensure there are deploy files for each environment. A list of environments can be taken from the prerequisites form.
+3. Compare the structure of the existing deploy files to what we have in [public demo shops](https://github.com/spryker-shop/b2c-demo-shop/blob/master/deploy.aws-env-template.yml), the structure has to be the same.
+4. Compare the section `Services` it has to be fitting default services as in the [example](https://github.com/spryker-shop/b2c-demo-shop/blob/master/deploy.aws-env-template.yml) (having less is fine).
```

Comment:
Do you mean that at least some of the default services should be defined?

Reply:
The list of services on a project has to match the list of services our Docker SDK has. In case the project has custom services, it won't be possible to use Docker SDK from our repository. Services must be replaced by ours or a custom Docker SDK fork must be used with regular merges of our Docker SDK into the fork.